presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -559,6 +559,14 @@ Procedures
 
     Create an empty partition in the specified table.
 
+* ``system.sync_partition_metadata(schema_name, table_name, mode)``
+
+    Check and repair partitions in metastore. There are three modes available:
+
+    * ``ADD`` : add any partitions that exist on the file system but not in the metastore.
```

Comment:
@electrum will it be obvious that user needs to put ADD in apostrophes?
or we could
```suggestion
    * ``'ADD'`` : add any partitions that exist on the file system but not in the metastore.
```
?

Reply:
Adding quotes will look a bit strange. We should add an example in the examples section below.